packages/tradershub/src/features/cfd/screens/MT5TradeScreen/MT5TradeDetailsItem/MT5TradeDetailsItem.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC, useRef } from 'react';
+import { useHover } from 'usehooks-ts';
+import { qtMerge, Text, useBreakpoint } from '@deriv/quill-design';
+import { Clipboard, Tooltip } from '../../../../../components/Base';
+import { useModal } from '../../../../../components/ModalProvider';
+import EditIcon from '../../../../../public/images/ic-edit.svg';
+
+type TMT5TradeDetailsItemProps = {
+    className?: string;
+    label?: string;
+    value: string;
+    variant?: 'clipboard' | 'info' | 'password';
+};
+
+const MT5TradeDetailsItem: FC<TMT5TradeDetailsItemProps> = ({ className, label, value, variant = 'clipboard' }) => {
+    const { isDesktop } = useBreakpoint();
+    const hoverRef = useRef(null);
+    const isHovered = useHover(hoverRef);
+    const { show } = useModal();
+    return (
+        <div
+            className={qtMerge(
+                'flex items-center h-1600 justify-between bg-system-light-secondary-background p-[5px] pl-400',
+                className
+            )}
+        >
+            {variant !== 'info' && (
+                <React.Fragment>
+                    <Text color='less-prominent' size='sm'>
+                        {label}
+                    </Text>
+                    <div className='flex items-center space-x-400 pr-400'>
```

Comment:
if you are using `flex` don't use `space-x-...` or `space-y-...` these utility classes can be use without `flex`. Use `gap` instead `gap-x-400` whenever you are using `flex` or `grid`